packages/local_auth/local_auth_darwin/darwin/Tests/FLALocalAuthPluginTests.m

```diff
@@ -49,11 +92,30 @@ - (void)setUp {
   self.continueAfterFailure = NO;
 }
 
+- (StubAlertFactory *)createStubAlertFactory {
+  StubAlertFactory *alertFactory;
+#if TARGET_OS_OSX
+
+  id mockNSAlert = OCMClassMock([NSAlert class]);
```

Comment:
Why is this a mock? The only thing that this appears to be used for in the test is to read a public property back out of it, which can be done with a real object, avoiding the need for a mock.